community/modules/scripts/ramble-execute/variables.tf

```diff
@@ -37,22 +75,36 @@ variable "commands" {
 }
 
 variable "ramble_runner" {
-  description = <<-EOT
-    Ansible based startup-script runner from a previous Ramble step
-    Will automatically be prepended to the script generated by this module.
-  EOT
+  description = "Runner from previous ramble-setup or ramble-execute to be chained with scripts generated by this module."
   default     = null
```

Comment:
If `ramble-execute: use: [ramble-setup] required` is true then I think this should have no default.